src/DocRaptor.php

```diff
@@ -176,26 +188,36 @@ public function fetchDocument($filename = false)
             }
 
             if (!empty($this->document_content)) {
-                $fields['doc[document_content]'] = urlencode($this->document_content);
+                $fields['doc[document_content]'] = $this->document_content;
             } else {
-                $fields['doc[document_url]'] = urlencode($this->document_url);
+                $fields['doc[document_url]'] = $this->document_url;
             }
 
-            $fields_string = http_build_query($fields);
+            $queryString = http_build_query($fields);
 
             $ch = curl_init();
-            curl_setopt($ch, CURLOPT_URL, $url);
+            curl_setopt($ch, CURLOPT_URL, $uri);
             curl_setopt($ch, CURLOPT_POST, count($fields));
-            curl_setopt($ch, CURLOPT_POSTFIELDS, $fields_string);
+            curl_setopt($ch, CURLOPT_POSTFIELDS, $queryString);
             curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
             $result = curl_exec($ch);
 
             if ($result) {
```

Comment:
Now kind of unhappy with how this `if` is gating all the stuff below it.

Wonder if it would be better to do the exception here like
```php
if (!$result) {
  throw new Exception('Curl error: ' . curl_error($ch));
}
```
And then the rest of the code is not indented and there's not this long-hanging else.